IFC4x3/Properties/l/LiningMaterial_0mAA1gW8z7mevSgdceBIp5/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LiningMaterial_0mAA1gW8z7mevSgdceBIp5" Name="LiningMaterial" UniqueId="3028a06a-808f-47c2-8e5c-aa79a82d2cc5" PrimaryDataType="IfcLabel" />
```

Comment:
Propose using `IfcPropertyReferenceValue.PropertyReference` : `IfcMaterialDefinition`

Reply:
Fixed in commit https://github.com/bSI-InfraRoom/IFC-Specification/pull/581/commits/f23b2d1e012d12e895e2730c791db8e953e79c53